contract/src/test/kotlin/io/provenance/scope/loan/contracts/RecordLoanContractUnitTest.kt

```diff
@@ -387,18 +387,6 @@ class RecordLoanContractUnitTest : WordSpec({
             }
         }
     }
-    "recordServicingRights" When {
```

Comment:
To be tested in `OverwriteServicingRightsUnitTest`, since we never merge the input with the existing record.